packages/css/test/__snapshots__/css.test.js.snap

```diff
@@ -83,8 +84,8 @@ exports[`css boolean as value 1`] = `
 exports[`css composition stuff 1`] = `
 .emotion-0 {
   -webkit-box-pack: center;
-  -webkit-justify-content: center;
   -ms-flex-pack: center;
+  -webkit-justify-content: center;
```

Comment:
the order of those properties has changed (its how they are generated now), I dont think it matters though